Application 8.10.0 crashes on start

The latest update causes a full crash upon start.

```
[kevin@kevinfedora ~]$ flatpak run com.axosoft.GitKraken 
Gtk-Message: 14:37:57.491: Failed to load module "canberra-gtk-module"
Gtk-Message: 14:37:57.491: Failed to load module "pk-gtk-module"
Gtk-Message: 14:37:57.492: Failed to load module "canberra-gtk-module"
Gtk-Message: 14:37:57.492: Failed to load module "pk-gtk-module"
[2:1011/143757.515500:ERROR:bus.cc(397)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:1011/143757.515582:ERROR:bus.cc(397)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
LaunchProcess: failed to execvp:
xdg-settings
[41:1011/143757.595305:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
[PERFORMANCE] tried to end event with no start time: waiting for app ready 
ldd: warning: you do not have execution permission for `/app/extra/gitkraken/resources/app.asar.unpacked/node_modules/@axosoft/nodegit/build/Release/nodegit-ubuntu-18.node'
Error: Cannot find module 'build/Debug/nodegit.node'
Require stack:
- /app/extra/gitkraken/resources/app.asar/node_modules/@axosoft/nodegit/lib/nodegit.js
- /app/extra/gitkraken/resources/app.asar/src/main/static/main.bundle.js
- 
    at Module._resolveFilename (node:internal/modules/cjs/loader:940:15)
    at Function.n._resolveFilename (node:electron/js2c/browser_init:249:1105)
    at Module._load (node:internal/modules/cjs/loader:785:27)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13343)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/app/extra/gitkraken/resources/app.asar/node_modules/@axosoft/nodegit/lib/nodegit.js:146:12)
    at Module._compile (node:internal/modules/cjs/loader:1116:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1169:10)
    at Module.load (node:internal/modules/cjs/loader:988:32)
    at Module._load (node:internal/modules/cjs/loader:829:12)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13343)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.16314 (/app/extra/gitkraken/resources/app.asar/src/main/static/main.bundle.js:2:4544050)
    at __webpack_require__ (/app/extra/gitkraken/resources/app.asar/src/main/static/main.bundle.js:2:4869593) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/app/extra/gitkraken/resources/app.asar/node_modules/@axosoft/nodegit/lib/nodegit.js',
    '/app/extra/gitkraken/resources/app.asar/src/main/static/main.bundle.js',
    undefined
  ]
}
```

## Triage

Single oneliner that somebody can use to install the previous version of GitKraken

`sudo flatpak update --commit=8d171075d5ff2a5ab7cf3946314bd06957e0f300fc360e65ae128a3a40646b08 com.axosoft.GitKraken`

```
[kevin@kevinfedora ~]$ flatpak run --talk-name=org.freedesktop.Flatpak com.axosoft.GitKraken
Gtk-Message: 15:22:22.967: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:22:22.968: Failed to load module "pk-gtk-module"
Gtk-Message: 15:22:22.968: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:22:22.968: Failed to load module "pk-gtk-module"
LaunchProcess: failed to execvp:
xdg-settings
[41:1011/152223.076937:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
[PERFORMANCE] tried to end event with no start time: waiting for app ready 
ldd: warning: you do not have execution permission for `/app/extra/gitkraken/resources/app.asar.unpacked/node_modules/@axosoft/nodegit/build/Release/nodegit-ubuntu-18.node'
Error: Cannot find module 'build/Debug/nodegit.node'
Require stack:
- /app/extra/gitkraken/resources/app.asar/node_modules/@axosoft/nodegit/lib/nodegit.js
- /app/extra/gitkraken/resources/app.asar/src/main/static/main.bundle.js
- 
    at Module._resolveFilename (node:internal/modules/cjs/loader:940:15)
    at Function.n._resolveFilename (node:electron/js2c/browser_init:249:1105)
    at Module._load (node:internal/modules/cjs/loader:785:27)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13343)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/app/extra/gitkraken/resources/app.asar/node_modules/@axosoft/nodegit/lib/nodegit.js:146:12)
    at Module._compile (node:internal/modules/cjs/loader:1116:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1169:10)
    at Module.load (node:internal/modules/cjs/loader:988:32)
    at Module._load (node:internal/modules/cjs/loader:829:12)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13343)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.16314 (/app/extra/gitkraken/resources/app.asar/src/main/static/main.bundle.js:2:4544050)
    at __webpack_require__ (/app/extra/gitkraken/resources/app.asar/src/main/static/main.bundle.js:2:4869593) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/app/extra/gitkraken/resources/app.asar/node_modules/@axosoft/nodegit/lib/nodegit.js',
    '/app/extra/gitkraken/resources/app.asar/src/main/static/main.bundle.js',
    undefined
  ]
}
```

Second machine here to test, same result. Interestingly, this machine is running the same distribution (Fedora 36) but this machine runs Wayland. So whatever the problem might be, it's both Wayland and X11.

## Output
```
[kevin@kevin-fedora-desktop ~]$ flatpak run com.axosoft.GitKraken 
[2 zypak-helper] Wait found events, but sd-event found none
Gtk-Message: 17:06:56.830: Failed to load module "canberra-gtk-module"
Gtk-Message: 17:06:56.831: Failed to load module "pk-gtk-module"
Gtk-Message: 17:06:56.831: Failed to load module "canberra-gtk-module"
Gtk-Message: 17:06:56.831: Failed to load module "pk-gtk-module"
[2:1011/170656.849097:ERROR:bus.cc(397)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:1011/170656.849159:ERROR:bus.cc(397)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
LaunchProcess: failed to execvp:
xdg-settings
[PERFORMANCE] tried to end event with no start time: waiting for app ready 
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[41:1011/170656.979814:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
ldd: warning: you do not have execution permission for `/app/extra/gitkraken/resources/app.asar.unpacked/node_modules/@axosoft/nodegit/build/Release/nodegit-ubuntu-18.node'
Error: Cannot find module 'build/Debug/nodegit.node'
Require stack:
- /app/extra/gitkraken/resources/app.asar/node_modules/@axosoft/nodegit/lib/nodegit.js
- /app/extra/gitkraken/resources/app.asar/src/main/static/main.bundle.js
- 
    at Module._resolveFilename (node:internal/modules/cjs/loader:940:15)
    at Function.n._resolveFilename (node:electron/js2c/browser_init:249:1105)
    at Module._load (node:internal/modules/cjs/loader:785:27)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13343)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/app/extra/gitkraken/resources/app.asar/node_modules/@axosoft/nodegit/lib/nodegit.js:146:12)
    at Module._compile (node:internal/modules/cjs/loader:1116:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1169:10)
    at Module.load (node:internal/modules/cjs/loader:988:32)
    at Module._load (node:internal/modules/cjs/loader:829:12)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13343)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.16314 (/app/extra/gitkraken/resources/app.asar/src/main/static/main.bundle.js:2:4544050)
    at __webpack_require__ (/app/extra/gitkraken/resources/app.asar/src/main/static/main.bundle.js:2:4869593) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/app/extra/gitkraken/resources/app.asar/node_modules/@axosoft/nodegit/lib/nodegit.js',
    '/app/extra/gitkraken/resources/app.asar/src/main/static/main.bundle.js',
    undefined
  ]
}
```

## Machine
```
System:
  Host: kevin-fedora-desktop Kernel: 5.19.14-200.fc36.x86_64 arch: x86_64
    bits: 64 Desktop: GNOME v: 42.5 Distro: Fedora release 36 (Thirty Six)
Machine:
  Type: Desktop Mobo: Micro-Star model: B450 GAMING PRO CARBON AC (MS-7B85)
    v: 1.0 serial: <superuser required> UEFI: American Megatrends LLC. v: 1.G0
    date: 07/25/2022
CPU:
  Info: 8-core model: AMD Ryzen 7 2700 bits: 64 type: MT MCP cache: L2: 4 MiB
  Speed (MHz): avg: 3457 min/max: N/A cores: 1: 3650 2: 3650 3: 2923
    4: 2931 5: 2929 6: 3650 7: 2938 8: 3650 9: 3650 10: 3650 11: 3650 12: 3650
    13: 3445 14: 3650 15: 3650 16: 3650
Graphics:
  Device-1: AMD Navi 22 [Radeon RX 6700/6700 XT/6750 XT / 6800M]
    driver: amdgpu v: kernel
  Display: wayland server: X.Org v: 1.22.1.3 with: Xwayland v: 22.1.3
    compositor: gnome-shell driver: X: loaded: amdgpu
    unloaded: fbdev,modesetting,radeon,vesa gpu: amdgpu resolution:
    1: 1280x1024~60Hz 2: 2560x1440~60Hz
  OpenGL: renderer: AMD Radeon RX 6700 XT (navy_flounder LLVM 14.0.0 DRM
    3.47 5.19.14-200.fc36.x86_64) v: 4.6 Mesa 22.1.7
Audio:
  Device-1: AMD Navi 21/23 HDMI/DP Audio driver: snd_hda_intel
  Device-2: AMD Family 17h HD Audio driver: snd_hda_intel
  Device-3: Cooler Master Sirus Headset type: USB
    driver: hid-generic,snd-usb-audio,usbhid
  Sound Server-1: ALSA v: k5.19.14-200.fc36.x86_64 running: yes
  Sound Server-2: PipeWire v: 0.3.59 running: yes
Network:
  Device-1: Intel Wireless-AC 9260 driver: iwlwifi
  IF: wlo1 state: down mac: 96:9b:03:6e:55:ee
  Device-2: Intel I211 Gigabit Network driver: igb
  IF: enp34s0 state: up speed: 1000 Mbps duplex: full
    mac: 00:d8:61:55:f1:dc
Bluetooth:
  Device-1: Intel Wireless-AC 9260 Bluetooth Adapter type: USB driver: btusb
  Report: rfkill ID: hci0 state: up address: see --recommends
Drives:
  Local Storage: total: 6.37 TiB used: 2.04 TiB (32.0%)
  ID-1: /dev/nvme0n1 vendor: Western Digital model: WDS100T2B0C-00PXH0
    size: 931.51 GiB
  ID-2: /dev/sda vendor: Seagate model: ST6000VN001-2BB186 size: 5.46 TiB
Partition:
  ID-1: / size: 925 GiB used: 385.3 GiB (41.7%) fs: btrfs dev: /dev/nvme0n1p3
  ID-2: /boot size: 3.86 GiB used: 298 MiB (7.5%) fs: ext4
    dev: /dev/nvme0n1p2
  ID-3: /boot/efi size: 2 GiB used: 13.9 MiB (0.7%) fs: vfat
    dev: /dev/nvme0n1p1
Swap:
  ID-1: swap-1 type: zram size: 8 GiB used: 0 KiB (0.0%) dev: /dev/zram0
Sensors:
  System Temperatures: cpu: 31.0 C mobo: 33.0 C gpu: amdgpu temp: 39.0 C
  Fan Speeds (RPM): fan-1: 0 fan-2: 1058 fan-3: 609 fan-4: 0 fan-5: 567
    fan-6: 547 gpu: amdgpu fan: 0
Info:
  Processes: 481 Uptime: 4m Memory: 31.27 GiB used: 3.94 GiB (12.6%)
  Shell: Bash inxi: 3.3.21
```

--- a/com.axosoft.GitKraken.json
+++ b/com.axosoft.GitKraken.json
@@ -216,8 +216,8 @@
                 {
                     "type": "file",
                     "dest-filename": "gitkraken.deb",
-                    "url": "https://release.axocdn.com/linux/GitKraken-v8.10.0.deb",
-                    "sha256": "6c4cd93c88986a6a5621c9d85c68bbf976547295638aa990f63992193934be2b",
+                    "url": "https://release.axocdn.com/linux/GitKraken-v8.9.1.deb",
+                    "sha256": "eb657b19f1bbca8fd923cbdfe651e9fd786ab73fa31ef34fcfba62ceebd09a10",
                     "x-checker-data": {
                         "type": "html",
                         "url": "https://www.gitkraken.com/download",
@@ -228,9 +228,9 @@
                 {
                     "type": "extra-data",
                     "filename": "gitkraken.deb",
-                    "url": "https://release.axocdn.com/linux/GitKraken-v8.10.0.deb",
-                    "sha256": "6c4cd93c88986a6a5621c9d85c68bbf976547295638aa990f63992193934be2b",
-                    "size": 92026876,
+                    "url": "https://release.axocdn.com/linux/GitKraken-v8.9.1.deb",
+                    "sha256": "eb657b19f1bbca8fd923cbdfe651e9fd786ab73fa31ef34fcfba62ceebd09a10",
+                    "size": 89176920,
                     "x-checker-data": {
                         "type": "html",
                         "url": "https://www.gitkraken.com/download",

--- a/com.axosoft.GitKraken.metainfo.xml
+++ b/com.axosoft.GitKraken.metainfo.xml
@@ -71,7 +71,6 @@
     <url type="help">https://www.gitkraken.com/contact</url>
 
     <releases>
-        <release version="8.10.0" date="2022-10-10"/>
         <release version="8.9.1" date="2022-09-14"/>
         <release version="8.9.0" date="2022-09-07"/>
         <release version="8.8.0" date="2022-08-10"/>

